service/build.gradle

```diff
@@ -164,3 +171,10 @@ bootJar {
     attributes('Implementation-Version': project.version.toString())
   }
 }
+sourceSets {
```

Comment:
Please remove, it works without it. that source set is configured by default in Gradle.